.vscode/settings.json

```diff
@@ -1,3 +1,5 @@
 {
-	"typescript.tsdk": "node_modules/typescript/lib"
-}
\ No newline at end of file
+	"typescript.tsdk": "node_modules/typescript/lib",
+	"editor.formatOnSave": false,
+	"editor.formatOnPaste": false
```

Comment:
Revert

Reply:
I erased this change from the commit tree.